spylib/utils/misc.py

```diff
@@ -22,19 +22,31 @@ def parse_scope(v: Any) -> List[str]:
     return v
 
 
+class _Time(BaseModel):
+    seconds: float
+    milliseconds: float
```

Comment:
This class is not internal because TimedResult is not internal to this module. Plus I've never seen _ used on a class name. I wouldn't recommend it.
Also in English the "time" is a point in time so not clear name.
Finally, leverage the class functionalities to make your life easier and the usage more intuitive, at least to me.

```suggestion
class ElapsedTime(BaseModel):
    start: float
    end: float
    
    @property
    def seconds(self) -> float:
        return self.end - self.start
    
    @property
    def milliseconds(self) -> float:
        return 1000 * self.end - self.start
```

Reply:
I thought what you meant by 
>You could make a time class with the time stored in _time
is make internal class. lol